### DISCUSSION TITLE
Imported ADMX file but no shortcuts created

### DISCUSSION THREAD

**Discussion by nortonrome** [Weatherlights/Winget-AutoUpdate-Intune#11] [General]
Hi, I deployed the Winget Autoupdate configurator to a machine via Intune. Afterwards I also added the newly created configuration profile including the admx file as described here. Intune now tells me that the profile is activated on my machine but even though I configured it in a way that it's supposed to create a startmenu/desktop shortcut I don't get one on the machine. Any possibility to debug this?

Thanks!!

**Comment by Weatherlights** [comment]
Have you checked the registry key HKEY_LOCAL_MACHINE\SOFTWARE\Policies\weatherlights.com\Winget-AutoUpdate if your configuration is applied.

If you already had a configuration in place earlier you should also enable "Reinstall on Policy Update" for shortcuts to be created when WAU in already installed.
